dashboard/app/views/pd/workshop_mailer/teacher_survey_reminder.html.haml

```diff
@@ -3,6 +3,8 @@
     font-family: Gotham, sans-serif;
   }
 
+-# This email is only for CSF Deep Dive (Pd::Workshop::SUBJECT_CSF_201)
+
 - if @workshop.subject == Pd::Workshop::SUBJECT_CSF_201
```

Comment:
I don't think you need the `if` anymore, since there isn't any other situation in which we send this email